### ISSUE TITLE
using Keras Lr Finder with Ernie 

### ISSUE THREAD

**Issue by Karim-Baig** [brunneis/ernie#20]
Thanks for this great package. 
While finding the optimal learning rate using Keras Lr Finder , how could this be incorporated. 
```
from keras_lr_finder import LRFinder
classifier=SentenceClassifier(model_name=Models.BertBaseUncased,max_length=256, labels_no=2)
classifier.load_dataset(train1,validation_split=0.1)
lr_finder = LRFinder(classifier)
lr_finder.find(classifier, 0.0001, 1, 5, 1)
```
This gives an error . Could you please suggest an alternative for using this or any other way for getting the optimal learning rate.
